cmd/query/app/query_parser.go

```diff
@@ -171,6 +171,7 @@ func (p *queryParser) parseTraceQueryParams(r *http.Request) (*traceQueryParamet
 			NumTraces:     limit,
 			DurationMin:   minDuration,
 			DurationMax:   maxDuration,
+			RawTraces:     false,
```

Comment:
@yurishkuro should this be set to false or do we want to read the form value here using `shouldUseRawTraces`? Previously, the `search` function would always perform the adjustment which is why I set this to false for now (https://github.com/jaegertracing/jaeger/blob/main/cmd/query/app/http_handler.go#L250)

Reply:
it should be standardized, and yes it should parse `raw` arg and set the value appropriately. You may want to move the parsing function into this type.